code/datums/storage/subtypes/others/carved_books.dm

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * Storage datums given to books upon getting carved out.
+ */
+
+/datum/storage/carved_book
+	max_slots = 1
+
+/datum/storage/carved_book/bible
+	max_specific_storage = WEIGHT_CLASS_SMALL
```

Comment:
You could move this to `misc.dm` instead. Creating a new file for just 10 lines of code seems overkill to me. Your call